tests/golos.referral_test_api.hpp

```diff
@@ -7,10 +7,10 @@ namespace eosio { namespace testing {
 class extended_tester : public golos_tester {
     using golos_tester::golos_tester;
     fc::microseconds _cur_time;
-    void update_cur_time() { _cur_time = control->head_block_time().time_since_epoch();};
 
 protected:
     const fc::microseconds& cur_time()const { return _cur_time; };
+    void update_cur_time() { _cur_time = control->head_block_time().time_since_epoch();};
```

Comment:
Why is this value cached? I do not see any reason to do so. Getting this value from the controller each time will not affect performance. But this solution will save the tests from the problem with the cached value.